File: test/cloud/gateway-server/src/main/java/io/github/zhangbinhub/acp/cloud/gateway/vo/ErrorVo.java
package io.github.zhangbinhub.acp.cloud.gateway.vo;

/**
 * @author zhang by 27/12/2018 13:07
 * @since JDK 11
 */
public class ErrorVo {

    private int code;
    private String error;
    private String errorDescription;

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }

    public String getErrorDescription() {
        return errorDescription;
    }

    public void setErrorDescription(String errorDescription) {
        this.errorDescription = errorDescription;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

}

File: boot/acp-spring-boot-starter/src/main/kotlin/pers/acp/spring/boot/vo/ErrorVo.kt
package pers.acp.spring.boot.vo

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * 错误信息实体
 *
 * @author zhang by 27/12/2018 13:07
 * @since JDK 11
 */
@ApiModel(value = "错误信息")
data class ErrorVo(
        @ApiModelProperty(value = "错误编码")
        var code: Int = 0,
        @ApiModelProperty(value = "信息")
        var error: String? = null,
        @ApiModelProperty(value = "描述")
        var errorDescription: String? = null
)
